packages/shared/src/components/filters/ContentTypesFilter.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from 'react';
+import { FilterCheckbox } from '../fields/FilterCheckbox';
+import { useAdvancedSettings } from '../../hooks/feed';
+import useFeedSettings from '../../hooks/useFeedSettings';
+
+export function ContentTypesFilter(): ReactElement {
+  const { advancedSettings, isLoading } = useFeedSettings();
+  const { selectedSettings, onToggleSettings } = useAdvancedSettings();
+  /*
+   * At the moment, we are only referencing the Video entity, but it should be based from the group that it is in.
+   * Point being, we have to send multiple mutation requests at the same time if the user toggles the switch.
+   * We should instead introduce a new mutation to handle an array of settings to toggle.
```

Comment:
Although for the others we already do individual mutations right?
The other advanced settings I mean.

Else you get that you need to toggle multiple and then press save somewhere.

Reply:
My point being: Wonder if we could already leverage group here and simply use the same mutation?
It's not needed for this implementation PR, but could probably be a very quick solution.

Reply:
I see, in this case, we will pass another parameter that would say toggle based on the group? And then tweak the mutation from the API to toggle all items under it.

Reply:
For now it's fine, not sure I understood it right.
It could be individual updates right?

Reply:
Yeah our endpoint supports individual updates but for this checkbox it needs to update all advanced settings under the group in talk.

Reply:
Why though? It's only video as being one setting no?
I'm a bit confused here.

Reply:
For now, there is one only `video:youtube`, but it can grow to more than one right? Let's say for example we will get another `video:vimeo` content type which will fall under the same category. This tab is designed to be the categories if I understand correctly.

Reply:
I think you turn `video` on/off not the specific types.
They are just subtypes for rendering purposes.

But let's fix it once we have another one.

Reply:
Yep, agreed, let's tackle it when it comes.